server/src/main/java/org/cloudfoundry/identity/uaa/provider/oauth/XOAuthAuthenticationFilter.java

```diff
@@ -89,6 +109,9 @@ private boolean authenticationWasSuccessful(
             ofNullable(successHandler).ifPresent(handler ->
                     handler.setSavedAccountOptionCookie(request, response, authentication)
             );
+        // TODO: :eyes_narrowed:
```

Comment:
We definitely need to remove this TODO before merging

Reply:
But why not fix it? Catching all exceptions scary.